sonar-helm-for-iac/main.go

```diff
@@ -18,34 +18,64 @@
 
 package main
 
-import "C"
 import (
+	"bufio"
+	"flag"
 	"fmt"
 	iac_helm "github.com/SonarSource/sonar-iac/sonar-helm-for-iac/org.sonarsource.iac.helm"
 	"google.golang.org/protobuf/proto"
+	"os"
 	"sigs.k8s.io/yaml"
 	"strings"
 	"text/template"
-	"unsafe"
 )
 
 func main() {
-	fmt.Println("Hello World!")
-}
+	var path string
+	var numTemplateLines int
+	flag.StringVar(&path, "path", "", "Name of the rawTemplate")
+	flag.IntVar(&numTemplateLines, "nl", 0, "Number of lines in the rawTemplate; lines after that are considered rawValues.yaml content")
```

Comment:
Generally speaking good idea of using stdin for passing the content of the files. However in current shape the mechanism is not extendable. What about the following solution:
Running the command without any arguments and passing by stdin information in following format:
```
templates/pod.yaml
42
<raw-data>
values.yaml
32
<raw-data>
END
```

In other words:
* name of the template
* number of bytes to read
* content of the file
* next files
* indicator that everything is passed

Thanks to that it will be easier later to pass other files without changing the interface. WDYT?

Reply:
Implemented in 57bde321148c4cfa186d26498cb346687e346ee4. You can take a look already; I'll address your other comments in a next commit